packages/ceramic-cli/src/index.ts

```diff
@@ -18,14 +25,39 @@ program
         if (stateStorePath == null) {
             stateStorePath = DEFAULT_PINNING_STORE_PATH
         }
-        await CeramicDaemon.create({
-            ipfsHost: ipfsApi,
+
+        const config = {
             ethereumRpcUrl: ethereumRpc,
             anchorServiceUrl: anchorServiceApi,
             stateStorePath: stateStorePath,
             validateDocs,
             pinning: pinning
-        })
+        }
+
+        if (ipfsApi) {
+            Object.assign(config, {
```

Comment:
Not an issue but seems this could simply be `config.ipfsHost = ipfsApi` as it's only setting this field, no?

Reply:
👍